cfme/tests/v2v/test_post_migrations.py

```diff
@@ -102,3 +116,7 @@ def test_migration_post_attribute(appliance, provider, mapping_data_vm_obj_mini,
     soft_assert(source_core == target_core)
     # Test7: memory on source and target vm
     soft_assert(source_memory == target_memory)
+    # Test8: check snapshot should be destroyed after migration
+    expected_count = 1 if provider.one_of(RHEVMProvider) else 0
```

Comment:
Why are we expecting 1 snapshot for RHV ?

Reply:
After migration of a VM to RHV, there is always a snapshot gets created to store its current status, however the same is not true in case of OSP.

Reply:
Please have a look to below comment
>> https://bugzilla.redhat.com/show_bug.cgi?id=1744563#c5

Reply:
Ok please add a comment about that

Reply:
Done